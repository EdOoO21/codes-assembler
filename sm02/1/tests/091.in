209 -647 981 400 389 -670 -177 943 -61 729
