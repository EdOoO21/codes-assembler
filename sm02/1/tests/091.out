729
-61
943
-177
-670
389
400
981
-647
209
